src/features/raid-bots/request-subcommand.ts

```diff
@@ -34,17 +35,22 @@ export class RequestSubcommand extends Subcommand {
         interaction.member?.roles as GuildMemberRoleManager
       );
 
-      await interaction.user.send(`You MUST add a message in https://discord.com/channels/539189546630381579/1129579809006178384 when you log onto a bot.
+      await interaction.user
```

Comment:
I think we can delete this. I removed the thread because folks didn't seem to like the idea. Can restore later if it seems desired.